tests/spec/core/biblio-spec.js

```diff
@@ -209,3 +209,27 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("should make sure references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `
+  <section class="informative" id="intro">
+    <h2>
```

Comment:
See if you can reduce this down to the absolute minimum of what is being tested (i.e., only section tag with the [dom] reference).